backend/TripBooker.Common/Hotel/Contract/HotelOccupationViewContract.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System;
+
+namespace TripBooker.Common.Hotel.Contract;
+
+public class HotelOccupationViewContract
+{
+    public Guid HotelId { get; set; }
+    public string HotelCode { get; set; } = null!;
+    public string HotelName { get; set; } = null!;
+    public string Country { get; set; } = null!;
+    public string AirportCode { get; set; } = null!;
+    public float Rating { get; set; }
+    public bool AllInclusive { get; set; }
```

Comment:
Dobrze rozumiem, że hotel albo jest all inclusive albo nie? Czy można potem wybrać dla hoteli, które mają to na true?

Reply:
Informacja o tym jest we view jeżeli o to ci chodzi

Reply:
Chodzi mi o to, czy w ramach jednego hotelu potem ktoś wybiera jakie chce wyżywienie?

Reply:
No tak to tylko informacja czy jest dostępne. Czy ktoś z tego korzysta to już będzie w rezerwacji

Reply:
A no to git